apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  __DEPRECATED__Text,
+  Button,
+} from "@webstudio-is/design-system";
+import type { Asset } from "@webstudio-is/prisma-client";
+import { getFormattedAspectRatio } from "./utils";
+import {
+  CloudIcon,
+  AspectRatioIcon,
+  SizeIcon,
+  TrashIcon,
+} from "@webstudio-is/icons";
+import { useSubmit } from "@remix-run/react";
+import prettyBytes from "pretty-bytes";
+import { Header } from "../../lib/header";
+
+const Filename = ({ name }: { name: string }) => {
+  const splitName = name.split(".");
+  const extension = splitName[splitName.length - 1];
+  const lastLetterBeforeExtension = splitName[splitName.length - 2]
+    .split("")
+    .pop();
+  return (
+    <Box
+      css={{
+        position: "relative",
+        maxWidth: "70%",
+      }}
+    >
+      <__DEPRECATED__Text
```

Comment:
can we please switch to the new Text component? I just made it so you don't have to in this  PR :)

Reply:
All done!